src/Dfc.CourseDirectory.Web/Controllers/ApprenticeshipsController.cs

```diff
@@ -74,54 +83,46 @@ public async Task<IActionResult> Search([FromQuery] SearchRequestModel requestMo
                 return RedirectToAction("Index", "Home", new { errmsg = "Please select a Provider." });
             }
 
-            ApprenticeshipsSearchResultModel model = new ApprenticeshipsSearchResultModel();
-
-            if (!string.IsNullOrEmpty(requestModel.SearchTerm))
+            if (string.IsNullOrWhiteSpace(requestModel?.SearchTerm))
             {
-                var result = await _apprenticeshipService.StandardsAndFrameworksSearch(requestModel.SearchTerm, UKPRN.Value);
+                return ViewComponent(nameof(ApprenticeshipSearchResult), new ApprenticeshipsSearchResultModel());
+            }
 
-                var listOfApprenticeships = new List<ApprenticeShipsSearchResultItemModel>();
+            var searchTermWords = GetSearchWords(requestModel.SearchTerm);
```

Comment:
I didn't really want to re-implement search, but it was here... https://github.com/SkillsFundingAgency/dfc-providerportal-apprenticeships/blob/main/src/Dfc.ProviderPortal.Apprenticeships/Helper/CosmosDbHelper.cs#L170-L222